### ISSUE TITLE
How to generate GT from 300W-LP

### ISSUE THREAD

**Issue by zero0kiriyu** [cleardusk/3DDFA#232]
Thank you for your great work! I am trying to reimplement this repository, but I noticed that you only provide processed training data. I would like to improve the performance on close eye data, so I need to know how to generate ground truth data for training. I tried to infer the preprocessing steps from 300W-LP and the labels you provided, but I couldn’t get it to work. Can you provide the detail how to generate the GT?  Thank you!

**Comment by zero0kiriyu** [comment]
just transform the t3d and scale, other parameter do not need transform
